webapp/src/app/query-runner/query-runner.component.css

```
@@ -11,6 +11,20 @@ button {
   width: 100%;
 }
 
+.mat-header-cell {
+  padding: 5px;
+  align-items: center;
+  overflow: hidden;
+  word-wrap: break-word;
+}
+
+.mat-cell {
+  padding: 5px;
+  align-items: center;
+  overflow: hidden;
+  word-wrap: break-word;
+}
```

You can merge both definitions
```suggestion
.mat-cell, .mat-header-cell {
  padding: 5px;
  align-items: center;
  overflow: hidden;
  word-wrap: break-word;
}
```